rectools/metrics/auc.py

```diff
@@ -339,25 +354,27 @@ def _get_sufficient_reco_explanation(self) -> str:
             not too high.
             """
 
-    def calc_per_user_from_fitted(self, fitted: AUCFitted) -> pd.Series:
+    def calc_per_user_from_fitted(self, fitted: AUCFitted, is_debiased: bool = False) -> pd.Series:
         """
         Calculate metric values for all users from from fitted data.
 
         Parameters
         ----------
         fitted : AUCFitted
             Meta data that got from `.fit` method.
+        is_debiased : bool, default False
+            If ``True``, indicator that a debias mechanism has been applied before.
+            If ``False``, indicator that the debias mechanism has not been applied before.
```

Comment:
```suggestion
            An indicator of whether the debias transformation has been applied before or not.
```

Reply:
I won't repeat it anymore, please fix everywhere

Reply:
Fixed